### ISSUE TITLE
Conditions (Dragonflight Boss Target Status)

### ISSUE THREAD

**Issue by DakHarding** [Dicebar/Raven#37]
One more issue I've found:

In Dragonflight dungeons and raid content "bosses" don't appear to trigger like other bosses from previous content when setting up a conditional under Target Status>Classification>Boss.

Thanks!

**Comment by Dicebar** [comment]
Interesting bug, thanks for reporting!

**Comment by Dicebar** [comment]
The [UnitClassification](https://wowpedia.fandom.com/wiki/API_UnitClassification) API call that this functionality relies on seems to, unfortunately, work as intended. In dungeons and raids the API says that bosses are "elites", despite their tooltips saying otherwise.

I suspect this is a bug on Blizzard's end.

We could try to make a workaround by generating lists of all bosses for all content and checking for IDs using [UnitGuid](https://wowpedia.fandom.com/wiki/API_UnitGUID), but that's something I'd rather avoid doing.

**Comment by DakHarding** [comment]
Yeah, adding ID's sound like a lot of unnecessary work. It's just strange because when, for example, running Time Walking dungeons or older raid content, the 'boss' trigger actually works fine. But none of the bosses register in Dragonflight. It's exhausting all the bugs/lag/connection issues I've seen in this xpac.

**Comment by Dicebar** [comment]
It's been excessive, especially with the introduction of Dragonflight UI code on classic. I've even seen a bug where the old version of Dragonflight UI code is disabled in Wrath but still works in DF 🤦
